ui/analyse/css/study/relay/_tour.scss

```diff
@@ -118,19 +123,16 @@
     }
 
     h1 {
-      flex: auto;
       @include fluid-size('font-size', 14px, 32px);
       color: $c-brag;
     }
 
     &__content {
       @extend %flex-column;
       flex: 1;
-      gap: 1em;
-      @include breakpoint($mq-col3) {
-        gap: calc(var(--box-padding) / 3);
-      }
-      margin: 1em 0;
+      justify-content: space-around;
+      padding: 0 2em;
+      min-height: 12vh;
```

Comment:
![291x677_2024-03-16](https://github.com/lichess-org/lila/assets/140370/b2d3674e-9b1b-4cef-b851-a6cf1552024d)

Reply:
that's why I went with gap and margin instead of space-around

Reply:
why is it showing an image in one and not the other?

Reply:
oh never mind, how does this render on prod?

Reply:
I seem to recall doing that because of the subscribe button i added (for some additional reason I can't remember). But subscribe's no longer there, and there's absolutely no reason to diverge from your properly working version. Please don't make me sync, resolve conflicts, all that again. Can you just revert my change in the merge, and if something comes up we can deal with it in another PR?